app/components/paperShow/components/collectionBox.scss

```diff
@@ -0,0 +1,375 @@
+@import "../../../variables";
+
+$color-mono-00: #1F2A36;
+$color-mono-01: #34495e;
+$color-mono-03: #77828C;
+$color-mono-04: #9AA7B5;
+$color-mono-06: #DAE1E8;
+$color-mono-07: #F0F1F2;
+$color-mono-08: #F7F9FA;
+$color-mono-09: #FAFBFC;
+$color-mono-10: #FDFEFF;
+
+$color-blue-01: #133F7A;
+$color-blue-03: #3e7fff;
+$color-blue-07: #E6EEFF;
+
+
+.fab {
+  position: fixed;
+  bottom: 24px;
+  right: 24px;
+  z-index: 100;
+
+  * {
+    margin: 0;
+    padding: 0;
+    font-family: "Roboto", sans-serif;
+    font-size: 14px;
+    line-height: 1.5;
+    color: $color-mono-00;
+    -webkit-font-smoothing: antialiased;        //anti-aliasing
+    text-shadow: 1px 1px 1px rgba(0,0,0,0.004); //anti-aliasing
+    outline: none;
+    ul {list-style: none;}
+  }
+
+  .paper_item_a {
+    padding: 12px 0;
+    &:first-child {
+      padding-top: 0;
+    }
+    &:not(:last-child) {
+      border-bottom: 1px solid $color-mono-06;
+    }
+    &:hover {
+      cursor: pointer;
+      .paper_item_a__paper__title {
+        color: $color-blue-03;
+      }
+    }
+    &__paper {
+      display: inline-block;
+      vertical-align: top;
+      width: 316px;
+      margin-right: 16px;
+      &__title {
+        width: 100%;
+        font-size: 14px;
+        font-weight: 500;
+        line-height: 1.4;
+        margin-bottom: 2px;
+        color: $color-blue-01;
+        transition: all ease 0.2s;
+      }
+      &__journal_authors {
+        width: 100%;
+        color: $color-mono-04;
+        font-size: 12px;
+      }
+    }
+    &__memo {
+      display: inline-block;
+      vertical-align: top;
+      width: 184px;
+      &__content {
+        font-size: 14px;
+        line-height: 1.4;
+        color: $color-mono-01;
+      }
+    }
+  }
+
+  .action_notification {
+    position: relative;
+    margin-bottom: 8px;
+    * {
+      font-family: "Roboto";
+    }
+    &__save {
+      padding: 8px 20px;
+      background-color: $color-mono-00;
+      border-radius: 9999px;
+      box-shadow: 2px 8px 8px 0px rgba(0,0,0, 0.25);
+      opacity: 0;
+      position: absolute;
+      right: 0;
+      bottom: 8px;
+      transform: translateX(100%);
+      transition: all ease 0.4s;
```

Comment:
I prefer to use explicit specific properties annotation for animation.
ex) transition:width 2s, height 2s, background-color 2s, transform 2s;
https://developer.mozilla.org/ko/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions